Resources/Prototypes/_Arc/Entities/clothing/eyes/arcglasses.yml

```diff
@@ -0,0 +1,91 @@
+#\Resources\Textures\_Arc\Clothing\eyes
+#for my ass not knowing what the directory for this shit
+
+- type: entity
+  parent: ClothingEyesBase
+  id: ClothingEyesGlassesMedical
+  name: medical glasses
+  description: Upgraded sunglasses that provide flash immunity and a medical HUD.
+  components:
+  - type: Sprite
+    sprite: _Arc/Clothing/eyes/medglasses.rsi
+  - type: Clothing
+    sprite: _Arc/Clothing/eyes/medglasses.rsi
+  - type: FlashImmunity
+  - type: EyeProtection
+    protectionTime: 5
+  - type: Tag
+    tags:
+    - HamsterWearable
+    - WhitelistChameleon
+    #- SecDogWearable # DeltaV - let Laika wear secglasses (no i dont wanna do another sprite shit for doggo)
+  - type: IdentityBlocker
+    coverage: EYES
+  - type: ShowHealthBars
+    damageContainers:
+    - Biological
+  - type: ShowHealthIcons
+    damageContainers:
+    - Biological
+
+- type: entity
+  parent: ClothingEyesBase
+  id: ClothingEyesGlassesMedSec
+  name: medsec glasses
+  description: Upgraded sunglasses that provide flash immunity and a medical HUD and security HUD.
+  components:
+  - type: Sprite
+    sprite: _Arc/Clothing/eyes/medsecglasses.rsi
+  - type: Clothing
+    sprite: _Arc/Clothing/eyes/medsecglasses.rsi
+  - type: FlashImmunity
+  - type: EyeProtection
+    protectionTime: 5
+  - type: Tag
+    tags:
+    - HamsterWearable
+    - WhitelistChameleon
+    #- SecDogWearable # DeltaV - let Laika wear secglasses (no i dont wanna do another sprite shit for doggo)
+  - type: IdentityBlocker
+    coverage: EYES
+  - type: ShowHealthBars
+    damageContainers:
+    - Biological
+  - type: ShowHealthIcons
+    damageContainers:
+    - Biological
+  - type: ShowJobIcons
+  - type: ShowMindShieldIcons
+  - type: ShowCriminalRecordIcons
+
+- type: entity
+  parent: ClothingEyesBase
+  id: ClothingEyesGlassesCR
+  name: agent glasses
+  description: Specialize glasses for agent in mission that provide flash and eyes protection and medical, security and diagnostic HUD.
```

Comment:
```suggestion
  description: Highly advanced glasses for Central Command personnel that provide flash immunity and all three HUD types.
```

Reply:
the reason why i dont mention central command so that the item wont be centcom only. it so that we can just reuse it for future content or something. 
tldr: i dont wanna add new glasses for each job that arent related to centcom that also need to have all hud thingi.

but il change it a bit cause agent in mission sound awful in my opinion

Reply:
also this thing are able to be use for welding hence eye protection.